app/src/components/main/analytics/events/sidebarFooterEvents.js

```diff
@@ -143,6 +143,16 @@ export const FOOTER_EVENTS = {
     action: 'Click on Documentation link',
     label: 'Transition to Documentation',
   },
+  POLICY_LINK: {
+    category: FOOTER,
+    action: 'Click on Privacy Policy link',
+    label: 'Transition to Privacy Policy',
+  },
+  NOTICE_LINK: {
+    category: FOOTER,
+    action: 'Click on Privacy Notice link',
+    label: 'Transition to Privacy Notice',
+  },
```

Comment:
Do we need to track these link clicks? Please clarify it with Data Analyst (Yulia Volkava).